src/main/webapp/amsterdam.html

```diff
@@ -45,11 +45,10 @@ <h1 class="mb-1" style="font-family: 'Pinyon Script', cursive; font-size:100px">
         </div>
       </div>
     </div>
-		<br><br>
+		
     <div class="container">
       <h1>Popular Landmarks!</h1>
       <p>Click on a landmark to see pictures of it!</p>
-  		<div id="map-container">
```

Comment:
I saw you still add css for map-container below, are you using it?

Reply:
No, the css is in red it was deleted also!

Reply:
Okk, I misread it.